CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.12](https://github.com/ledgerhq/app-exchange/compare/2.0.11...2.0.12) - 2022-??-??
+
+### Changed
+
+- Application name length is no longer caped to 15B, but synced with BOLOS maximum length
+
+## [2.0.11](https://github.com/ledgerhq/app-exchange/compare/2.0.3...2.0.11) - 2022-05-24
+
+TODO
```

Comment:
To do !

Reply:
Frankly, that'll take much longer that what I'm ready to offer to exchange rn.